test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests/Get_ApplicationJson_ArrayObject_Tests.cs

```diff
@@ -15,7 +15,7 @@ namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.Document.Tests
     public class Get_ApplicationJson_ArrayObject_Tests
     {
         private static HttpClient http = new HttpClient();
-
+        var test = 0;

```

Comment:
This `var` keyword cannot be used for fields. Could you please have a look again?